src/js/View.js

```diff
@@ -425,12 +425,12 @@ View = (function() {
             else if (view.mode==View.SELECT) {
                 view.selectStartCoo = {x: view.dragx, y: view.dragy};
             }
-            return false; // to disable text selection
+            return true; // false disables default browser behaviour like possibility to touch hold for context menu. To disable text selection use css user-select: none instead of putting this value to false
```

Comment:
The comment describes the first problem quite well. Returning false here entirely disables a default behavior of browsers: context menus. i.e. when a user right-clicks anything (or in case it is a touch device a context menu is opened by touch and hold).
According to the original comment "to disable text selection" the reason for returning false is to disable text selection.  This desired behavior can be achieved through css instead with user-select: none. 

In the case of ESASky, we have created our own context menu, which is shown when the user performs the context menu action (i.e right-click on devices with mouse, touch hold on touch devices, potentially other ways to open context menus on other devices)